_posts/2023-07-20-alert-anomaly-visual.md

```diff
@@ -0,0 +1,132 @@
+##### Figure 5. Visualization showing Add alerting monitor context menu
+
+3. Choose **Create new monitor** or **Associate existing monitor**.
+- If you choose to create a new monitor, input the required information under **Monitor details**.
+- If you choose to associate an existing monitor, select a monitor from the dropdown list under **Select monitor to associate**.
+
+4. Select **Create monitor** or **Associate monitor**, as applicable, to add the monitor to your visualization, as shown in the following images.
+
+| Create monitor | Associate monitor |
+|:--------------:|:-----------------:|
+| ![Create monitor interface]({{site.url}}{{site.baseurl}}/assets/media/blog-images/2023-07-20-alert-anomaly-visual.md/save-new-monitor.png) | ![Associate monitor interface]({{site.url}}{{site.baseurl}}/assets/media/blog-images/2023-07-20-alert-anomaly-visual.md/save-associated-monitor.png) |
+
+### Viewing events
+
+Regardless of whether you choose to create a new monitor or detector or to associate an existing monitor or detector, events on your dashboard are triggered based on the criteria you set up. For a detailed view of the trigger criteria, go to the **View Events** page. You have two options for accessing the **View Events** menu: Select an event on the visualization itself or select **View Events** from the visualization’s context menu, as shown in the following image.
+
+![View events context menu interface]({{site.url}}{{site.baseurl}}/assets/media/blog-images/2023-07-20-alert-anomaly-visual.md/view-events-context-menu.png)
+##### Figure 6. Visualization showing View events context menu
+
+As shown in the following image, the events flyout has two sections. The top section is the visualization displaying the same aggregate view of events and time range as the visualization on the dashboard. The events flyout can be refreshed to fetch any new data. The bottom section displays the specific event details. This section is arranged by plugin (Anomaly Detection or Alerting), then by the individual plugin resources within that plugin (anomaly detectors or alerting monitors). In this view, you can see which plugin resources are producing events and when those events were produced. An overall event count for each plugin resource is displayed next to the resource name. To display more information about a specific plugin resource or the event it has produced, select the resource name. This opens a new tab that displays the resource details about that plugin. A time range is also fixed at the bottom of the flyout to help visually correlate individual events.
+
+![View events flyout interface]({{site.url}}{{site.baseurl}}/assets/media/blog-images/2023-07-20-alert-anomaly-visual.md/events-flyout.png)
+##### Figure 7. Events flyout showing alerting and anomaly detectors details and a line chart visualization
+
+## Limitations
+
+Currently, this feature has following limitations:
```

Comment:
The following is a list of things we want to work more on, but didn't get to...

We need to add that we didn't get to building out all the new components as common components to be used by others. We plan to do that in a follow up release.